java/src/main/java/com/powsybl/dataframe/network/extensions/CgmesMetadataModelDataframeAdder.java

```diff
@@ -0,0 +1,81 @@
+import com.powsybl.dataframe.network.adders.AbstractSimpleAdder;
+import com.powsybl.dataframe.network.adders.SeriesUtils;
+import com.powsybl.dataframe.update.IntSeries;
+import com.powsybl.dataframe.update.StringSeries;
+import com.powsybl.dataframe.update.UpdatingDataframe;
+import com.powsybl.iidm.network.*;
+
+import java.util.*;
+
+/**
+ * @author Naledi El Cheikh <naledi.elcheikh@rte-france.com>
+ */
+
+public class CgmesMetadataModelDataframeAdder extends AbstractSimpleAdder {
+    private static final List<SeriesMetadata> METADATA = List.of(
+            SeriesMetadata.stringIndex("id"),
+            SeriesMetadata.strings("subset"),
+            SeriesMetadata.strings("description"),
+            SeriesMetadata.ints("version"),
+            SeriesMetadata.strings("modelingAuthoritySet"),
+            SeriesMetadata.strings("profiles"),
+            SeriesMetadata.strings("dependentOn"),
+            SeriesMetadata.strings("supersedes")
+    );
+
+    @Override
+    public List<List<SeriesMetadata>> getMetadata() {
+        return Collections.singletonList(METADATA);
+    }
+
+    private static final class CgmesMetadataSeries {
+        private final StringSeries id;
+        private final StringSeries subset;
+        private final StringSeries description;
+        private final IntSeries version;
+        private final StringSeries modelingAuthoritySet;
+        private final StringSeries profiles;
+        private final StringSeries dependentOn;
+        private final StringSeries supersedes;
+
+        private CgmesMetadataSeries(UpdatingDataframe dataframe) {
+            this.id = dataframe.getStrings("id");
+            this.subset = dataframe.getStrings("subset");
+            this.description = dataframe.getStrings("description");
+            this.version = dataframe.getInts("version");
+            this.modelingAuthoritySet = dataframe.getStrings("modelingAuthoritySet");
+            this.profiles = dataframe.getStrings("profiles");
+            dependentOn = dataframe.getStrings("dependentOn");
+            this.supersedes = dataframe.getStrings("supersedes");
+        }
+
+        void create(int row, CgmesMetadataModelsAdder adder) {
+            CgmesMetadataModelsAdder.ModelAdder modelAdder = adder.newModel();
+            SeriesUtils.applyIfPresent(id, row, modelAdder::setId);
+            SeriesUtils.applyIfPresent(subset, row, subset -> modelAdder.setSubset(CgmesSubset.valueOf(subset)));
+            SeriesUtils.applyIfPresent(description, row, modelAdder::setDescription);
+            SeriesUtils.applyIfPresent(version, row, modelAdder::setVersion);
+            SeriesUtils.applyIfPresent(modelingAuthoritySet, row, modelAdder::setModelingAuthoritySet);
+            SeriesUtils.applyIfPresent(profiles, row, modelAdder::addProfile);
+            SeriesUtils.applyIfPresent(dependentOn, row, modelAdder::addDependentOn);
+            SeriesUtils.applyIfPresent(supersedes, row, modelAdder::addSupersedes);
```

Comment:
The methods `modelAdder::add...` should be replaced with a method that
- takes the 'element1/element2/...' string as an input
- parse it as a list containing the different elements
- calls  `modelAdder::add...` for every element (adding them to the corresponding set)